specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/StartStopManagedInstanceSchedules.json

```diff
@@ -0,0 +1,389 @@
+          },
+          "204": {
+            "description": "The specified Start/Stop schedule does not exist."
+          }
+        },
+        "x-ms-examples": {
+          "Deletes the managed instance's Start/Stop schedule.": {
+            "$ref": "./examples/StartStopManagedInstanceScheduleDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScheduleItem": {
+      "description": "Schedule info describing when the server should be started or stopped.",
+      "required": [
+        "startDay",
+        "startTime",
+        "stopDay",
+        "stopTime"
+      ],
+      "type": "object",
+      "properties": {
+        "startDay": {
+          "description": "Start day.",
+          "enum": [
+            "Sunday",
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "DayOfWeek",
+            "modelAsString": true
+          }
+        },
+        "startTime": {
+          "description": "Start time.",
+          "type": "string"
+        },
+        "stopDay": {
+          "description": "Stop day.",
+          "enum": [
+            "Sunday",
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "DayOfWeek",
+            "modelAsString": true
+          }
+        },
+        "stopTime": {
+          "description": "Stop time.",
+          "type": "string"
+        }
+      }
+    },
+    "StartStopManagedInstanceSchedule": {
+      "description": "Managed instance's Start/Stop schedule.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "systemData": {
+          "$ref": "../../../common/v1/types.json#/definitions/systemData",
+          "description": "System data of the scheduled resource.",
+          "readOnly": true
+        },
```

Comment:
If you used v3 types then systemdata is already present there.

Reply:
Hi @raosuhas for the common types v3/v4, we have had the exception from the ARM side for this version.
https://github.com/Azure/azure-rest-api-specs/pull/21415#discussion_r1026792600